src/lib/CMakeLists.txt

```diff
@@ -413,6 +413,8 @@ set(
     storage/index/group_key/variable_length_key_store.cpp
     storage/index/group_key/variable_length_key_store.hpp
     storage/index/index_info.hpp
+    storage/index/counting_quotient_filter/counting_quotient_filter.hpp
+    storage/index/counting_quotient_filter/counting_quotient_filter.cpp
```

Comment:
What is the reason to have the CQF integrated as an index instead of a filter?
I’d thought the index interface is a less good fit, right?

Reply:
Should that go into statistics?